### DISCUSSION TITLE
Corpora?

### DISCUSSION THREAD

**Discussion by kleer001** [adieyal/sd-dynamic-prompts#124] [Ideas]
These folders of tags in the Wildcard Files of Jumbo remind me of : https://github.com/dariusk/corpora/tree/master/data

"This project is a collection of static corpora (plural of "corpus") that are potentially useful in the creation of weird internet stuff."

I bet there'd be a lot of useless tags, but an opportunity for a massive library. It has a CC0 license. 

Is that what was started with?

**Comment by adieyal** [comment]
Thanks for the link - I haven't come across it before. I'll go through it and add anything that looks useful
